internal/runes/runes.go

```diff
@@ -0,0 +1,24 @@
+package runes
+
+import "github.com/samber/lo"
+
+// IndexNotByte is similar with strings.IndexByte but showing the opposite behavior.
```

Comment:
I would find it easier to just describe the actual behavior. I'm not entirely sure what to expect from "opposite of index-of", but I'd go with "find the first index that is _not_ `c`" -- I can't map this understanding to the implementation, though.

Reply:
Yeah, this is a nonsense function description lol. This was copied from go-prompt's similar implementation but I updated it